draft-ietf-quic-v2.md

```diff
@@ -200,15 +200,17 @@ The client MUST NOT send 0-RTT packets using the negotiated version, even after
 processing a packet of that version from the server. Servers can apply original
 version 0-RTT packets to a connection without additional considerations.
 
-# TLS Resumption
+# TLS Resumption and NEW_TOKEN tokens
 
-TLS session tickets are specific to the QUIC version of the connection that
-provided them. Clients MUST NOT use a session ticket from a QUICv1 connection
-to initiate a QUICv2 connection, or vice versa.
+TLS session tickets and NEW_TOKEN tokens are specific to the QUIC version of the
+connection that provided them. Clients MUST NOT use a session ticket or token
+from a QUICv1 connection to initiate a QUICv2 connection, or vice versa.
```

Comment:
I think that I might prefer a different polarity on the SHOULD and MUST clauses in this section.  That is, servers MUST NOT and clients SHOULD.  The point here being that a malicious client will produce a ticket from the wrong version and the server is ultimately responsible for ensuring that it doesn't result in badness.